apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -49,7 +49,7 @@ export default class DetailViewContents extends React.Component {
 
   state = {
     status: this.props.applicationData.status,
-    notes: this.props.applicationData.notes
+    notes: this.props.applicationData.notes || "Google doc rubric completed: Y/N\nTotal points:\n(If interviewing) Interview notes completed: Y/N\nAdditional notes:"
```

Comment:
Do we need this here? Won't the [model](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/application/application_base.rb#L86) provide that template text?

Reply:
this has changed and is no longer being provided by the model. It is just the initial value of the text field in the UI and the user can edit and save it or delete it.

Reply:
I'm actually okay with the template existing only server-side

Reply:
We decided not to stick with the original plan because when you viewed the quick view table, it had the template text for all applications and was hard to tell if the notes had been edited or not.